ffi/capi/c/examples/fixeddecimal_tiny/icu_capi_staticlib_tiny/Cargo.toml

```diff
@@ -15,11 +15,4 @@ crate-type = ["staticlib"]
 icu_capi = { path = "../../../..", default-features = false, features = ["compiled_data", "icu_decimal"] }
 dlmalloc = { version = "0.2", features = ["global"] }
 
-[profile.release]
-lto = true
-
-[profile.release-opt-size]
-inherits = "release"
-opt-level = "s"
```

Comment:
Observation: these settings are already in the examples directory and therefore reproducible.

Reply:
Not for tinywasm